content/blog/entries/why-lektor/contents.lr

```diff
@@ -0,0 +1,76 @@
+title: Why Lektor?
+---
+categories:
+outreachy
+tech
+open-source
+---
+author: dhruvi16
+---
+pub_date: 2020-05-13
+---
+body:
+
+
+<div style="text-align: center;">
+  <figure>
+    <img src="cover.jpeg"/>
+    <figcaption>
+      <em>
+        why?why?why?
+      </em>
+    </figcaption>
+  </figure>
+</div>
+
+I wrote this article while prepping for my Outreachy internship. I was trying to find out why Lektor is being used for the project and found out really convincing reasons for the same.
+
+Let’s start with basic types of website -
+
+Static website: Here the webpages are pre-loaded to the server and when the request is made, these pages are served to the client as it is. These webpages can use HTML, CSS, and Javascript. By static, it does not mean that these webpages are devoid of user interactivity. These pages can be highly reactive (because of course JS) but yes they are not generated on the server and are static in that manner.
+
+<div style="text-align: center;">
+  <figure>
+    <img src="static.jpg"/>
+    <figcaption>
+      <em>
+        Static Website.
+      </em>
+    </figcaption>
+  </figure>
+</div>
```

Comment:
If you use an image from a different source, like Tutorials Point in this case, it would be a good idea to 
- prefer CC licensed images, because of course
- provide full attribution as per the requirements of the source website
Tutorials Point content seems to be copyrighted so you could consider replacing these images altogether.